packages/@aws-cdk/aws-appmesh/lib/shared-interfaces.ts

```diff
@@ -194,3 +196,32 @@ class FileAccessLog extends AccessLog {
   }
 }
 
+/**
+ * Represents the properties needed to define a backend
+ */
+export interface Backend {
```

Comment:
I'm still partial to leaving this as a `bind()` class. We could have other backend types in the future

Reply:
I couldn't imagine any fundamentally different outbound traffic (other backend types). I could only think of additional handling logic for the outbound traffic (client side throttling), which would be implemented by adding properties to the backend.

Also the `backends` property in the [API is a list](https://docs.aws.amazon.com/app-mesh/latest/APIReference/API_VirtualNodeSpec.html#appmesh-Type-VirtualNodeSpec-backends) and I don't think that leaves room to create other backend types in the future.

Reply:
The `backend` property is [not a list though](https://docs.aws.amazon.com/app-mesh/latest/APIReference/API_Backend.html). We could easily introduce a different type here in the future, such as using a Virtual Gateway as a target or some sort of egress type.